tests/lib/cli.js

```diff
@@ -54,10 +54,12 @@ describe("cli", () => {
      */
     async function verifyESLintOpts(cmd, opts, configType) {
 
+        const ActiveESLint = configType === "flat" ? FlatESLint : ESLint;
```

Comment:
Currently, `configType` is always undefined when this function (`verifyESLintOpts`) is called, so `ActiveESLint` will be always the legacy `ESLint`.

Reply:
Looks like this helper function is used only in two eslintrc-only tests.